app/services/idv/steps/send_link_step.rb

```diff
@@ -28,8 +28,22 @@ def call
         build_telephony_form_response(telephony_result)
       end
 
+      def extra_view_variables
+        {
+          idv_phone_form: create_form,
+        }
+      end
```

Comment:
This feels kind of yucky, but I think is the way to do this until we pull the hybrid flow out of the FSM?

Reply:
Yeah, I think if we were invested in improving FSM, it could be nice to somehow provide a form object through the FSM machinery, but it seems like we're moving away from that, and this is the otherwise standard way to make values available to the view, so seems fine to me.